next/components/molecules/DocumentGroup.tsx

```diff
@@ -16,7 +19,12 @@ const DocumentGroup = ({ documents }: DocumentGroupFragment) => {
   }, [documents])
 
   return (
-    <div className="flex flex-col gap-4">
+    <div
+      className={cx('flex flex-col gap-4', {
+        'divide-2 gap-0 divide-y divide-solid divide-[#d3d3d3] divide-[1px]':
```

Comment:
You have two "size" clasess `divide-2 divide-[1px]` -> use only `divide-px`
`divide-solid` should not be needed I guess